style.css

```diff
@@ -648,7 +599,111 @@ textarea::placeholder {
 
 /* side menu  */
 
+/* popup  */
+.modal-container {
+  position: fixed;
+  height: 100vh;
+  width: 100vw;
+  backdrop-filter: blur(15px);
+  background-color: rgb(193, 199, 208, 0.65);
+  z-index: 100;
+  display: none;
+  justify-content: center;
+  align-items: center;
+}
+
+.modal {
+  width: auto;
+  max-width: 1048px;
+  background-color: #fff;
+  padding: 0 20px 20px 20px;
+  margin: 20px;
+  border-radius: 16px;
+  transform: scale(0);
+  transition: all ease-in-out 0.5s;
+  position: relative;
+}
+
+.open-modal {
+  transform: scale(1);
+  transition: all ease-in-out 0.5s;
+}
+
+.open-modal-container {
+  display: flex;
+}
+
+.modal-image img {
+  width: 100%;
+}
+
+.modal-image {
+  width: 100%;
+  height: 90vh;
+  max-height: 220px;
+  max-width: 1000px;
+  background-position: center center;
+  background-size: 100%;
+  background-repeat: no-repeat;
+}
+
+.modal-title {
+  margin-bottom: 0;
+}
+
+.modal-highlights {
+  margin-bottom: 20px;
+}
+
+.modal-buttons {
+  display: flex;
+  justify-content: space-between;
+  width: 190px;
+}
```

Comment:
[OPTINAL]
- [ ] Kindly consider adding a line at the top of the modal buttons and make the buttons a little bigger to follow the Figma design

### Your design :point_down: 
![Screenshot from 2023-04-14 02-47-13](https://user-images.githubusercontent.com/91142432/231929059-8141bcfc-3800-434a-8a9a-85392e2f5a7c.png)

### What it's supposed to look like :point_down: 
![231780398-db83a483-30c3-4757-a929-c3aba04ce48d](https://user-images.githubusercontent.com/91142432/231929198-573582d3-ee7c-4953-8f4c-b4b6ec01b2d6.png)